introduction/problems-solved.md

```diff
@@ -0,0 +1,35 @@
+This file holds the text to accompany the [What Problems Does InnerSource Solve?](https://www.safaribooksonline.com/videos/introduction-to-innersource/9781492041504/9781492041504-video321607) video.
+This accompanying text will appear as a part of the learning path (like [this example](https://www.safaribooksonline.com/learning-paths/learning-path-lean/9781491999738/9781491946527-/part01ch01.html)).
+
+# What Problems Does InnerSource Solve?
+
+To illustrate the types of situations where inner source can help, imagine the following.
```

Comment:
I feel that InnerSource solves problems above and beyond the situation you described in this article. On a more abstract level, it fosters collaboration and learning where both was not possible beforehand due to strong siloing. The situation you describe here is "merely" one symptom of that problem. How about we lead with a more general statement along the lines of what I mentioned above and then follow with the concrete example you describe here.

Reply:
Great idea!  Added an extra paragraph at the beginning.